app/src/components/common/tokens/index.tsx

```diff
@@ -29,8 +31,24 @@ export const Tokens = (props: Props) => {
       value: id,
     }))
 
+  const isTokenInKnowTokens = tokens.find(token => token.label === value.symbol)
```

Comment:
If we merge `Collateral` and `Token`, and make a `getKnownTokens(networkId)` function, then maybe we can drop all of this and just do something like this above:

```typescript
const tokens = Object.entries([...getKnownTokens(networkId), ...customValues])
```